sxiv.h

```diff
@@ -116,6 +116,14 @@ typedef enum {
 	FF_TN_INIT = 4
 } fileflags_t;
 
+typedef enum {
+	BASE_CFILE,
+	CFILE,
+	EMPTY,
+
+	SUFFIXMODE_COUNT,
```

Comment:
What's the purpose of `SUFFIXMODE_COUNT`?
Also there's an empty line here. And the rest of the codebase **doesn't** use comma on the last enum.